tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -3154,6 +3154,22 @@ public function userSetTheMessageExpirationToXWithStatusCode(string $user, int $
 		$this->assertStatusCode($this->response, $statusCode);
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" set can mention everyone to (all|moderators) of room "([^"]*)" with (\d+)(?: \((v4)\))?$/
+	 */
+	public function userSetMentionEveryoneOfRoomWithStatus(string $user, string $who, string $identifier, int $statusCode, string $apiVersion = 'v1'): void {
+		if ($who === 'all') {
+			$permission = 1;
+		} elseif ($who === 'moderators') {
+			$permission = 0;
+		}
+		$this->setCurrentUser($user);
+		$this->sendRequest('PUT', '/apps/spreed/api/' .  $apiVersion . '/room/' . self::$identifierToToken[$identifier] . '/can-mention-everyone', [
+			'permission' => $permission,
```

Comment:
```suggestion
		if ($who === 'all') {
			$config = 1;
		} elseif ($who === 'moderators') {
			$config = 0;
		}
		$this->setCurrentUser($user);
		$this->sendRequest('PUT', '/apps/spreed/api/' .  $apiVersion . '/room/' . self::$identifierToToken[$identifier] . '/can-mention-everyone', [
			'config' => $config,
```


as per suggestion in the docs file